src/endpoints/getPlayerStats.ts

```diff
@@ -0,0 +1,59 @@
+import FullPlayerStats from '../models/FullPlayerStats'
+import Team from '../models/Team'
+import HLTVConfig from '../models/HLTVConfig'
+import { fetchPage } from '../utils/mappers'
+
+const getPlayerStats = (config: HLTVConfig) => async ({ id , startDate, endDate }: { id: number, startDate: string, endDate: string }): Promise<FullPlayerStats> => {
+    
+
+    var options = '';
+    if (startDate != null && endDate != null) {
+        options = '?startDate='+startDate+'&endDate='+endDate;
+    }
+    const $ = await fetchPage(`${config.hltvUrl}/stats/players/${id}/-${options}`, config.loadPage)
+
+    const name = $('.statsPlayerName').text() || undefined
+    const ign = $('.context-item-name').text();
+
+    const image = $('.context-item-image').attr('src');
+
+    const getInfo = function(i) { return $($('.divided-row').find('.large-strong').get(i)); };
+    const age = Number(getInfo(1).text()) || undefined;
+
+    const country = {
+        name: getInfo(2).text(),
+        code: undefined
+    }
+
+    let team: Team | undefined
+    
+    if (getInfo(3).text().trim() !== '-') {
+        team = {
+            name: getInfo(3).text(),
+            id: Number(getInfo(3).attr('href').split('/')[3]),
+            image: $($('.image-and-label>img').get(1)).attr('src')
```

Comment:
Please remove the `image` property from the object. You can scroll to the bottom of the page for the explanation.